src/olcs/VectorSynchronizer.ts

```diff
@@ -79,27 +66,22 @@ class VectorSynchronizer extends olcsAbstractSynchronizer {
   /**
    * Synchronizes the layer visibility properties
    * to the given Cesium Primitive.
-   * @param {import('olsc/core.js').LayerWithParents} olLayerWithParents
-   * @param {!Cesium.Primitive} csPrimitive
    */
-  updateLayerVisibility(olLayerWithParents, csPrimitive) {
+  updateLayerVisibility(olLayerWithParents: LayerWithParents, csPrimitive: PrimitiveCollection) {
     let visible = true;
     [olLayerWithParents.layer].concat(olLayerWithParents.parents).forEach((olLayer) => {
       const layerVisible = olLayer.getVisible();
       if (layerVisible !== undefined) {
-        visible &= layerVisible;
+        visible = visible && layerVisible;
```

Comment:
Thanks for the fix. This is also possible:
```suggestion
        visible &&= layerVisible;
```